app/services/bookmark_export_service.rb

```diff
@@ -106,7 +106,10 @@ def blackboard_export(data)
   #
   # @return [Hash] A hash containing the export data, filename, and content type
   def d2l_export(data)
-    export_result(data:)
+    export_result(data:,
+                  filename: file_name(suffix: 'zip'),
```

Comment:
if we are always outputting a zip, can we change the values for suffix and type in the formatter?

Reply:
@laritakr This led us to a rabbit hole and we refactored a bit, please see second commit